src/Database/Database.php

```diff
@@ -519,7 +517,7 @@ public function createCollection(string $id, array $attributes = [], array $inde
     {
         $collection = $this->silent(fn() => $this->getCollection($id));
         if (!$collection->isEmpty() && $id !== self::METADATA){
-            throw new Duplicate('Collection ' . $id . ' Exists!');
+            throw new DuplicateException('Collection ' . $id . ' Exists!');
```

Comment:
This is okay 👍 

Apart from this, the other changes in the file are not required. `filters` are not something an end user can fix from the dashboard